Source/Core/VideoCommon/RenderBase.h

```diff
@@ -62,7 +62,8 @@ enum class OSDMessage : s32
   EFBCopyToggled = 3,
   FogToggled = 4,
   SpeedChanged = 5,
-  XFBChanged = 6
+  XFBChanged = 6,
+  VolumeChanged = 7
```

Comment:
Please add a trailing comma, so that future diffs are shorter and clearer when someone decides to add another enum value.